app/api/v2/handlers/base_object_api.py

```diff
@@ -142,9 +142,9 @@ async def delete_on_disk_object(self, request: web.Request):
 
     """Helpers"""
 
-    async def _parse_common_data_from_request(self, request) -> (dict, dict, str, dict, dict):
+    async def _parse_common_data_from_request(self, request, read_body) -> (dict, dict, str, dict, dict):
         data = {}
-        if request.has_body:
+        if read_body:
```

Comment:
Made these changes as there seem to be some version issues with aiohttp that the UI team was running into. Some aiohttp methods work for them and not for me, and vice versa. Was thinking that looking into this issue could be a separate ticket, but at the very least, this change should work for all versions.